exercise1/Program.cs

```diff
@@ -0,0 +1,105 @@
+﻿using System;
+using Newtonsoft.Json;
+using RestSharp;
+using Retrofit.Net;
+
+namespace exercise1
+{
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var dataCenter = "us17";
```

Comment:
The program should able to enter an email, an email content from user and send the email.

Reply:
done